src/librbd/journal/Types.h

```diff
@@ -140,13 +140,17 @@ struct OpFinishEvent : public OpEventBase {
 };
 
 struct SnapEventBase : public OpEventBase {
+  cls::rbd::SnapshotNamespace snap_namespace;
```

Comment:
Need to add encode/decode of this to all derived events (you can't add it to the base class w/o breaking encoding). You can put the dump in the base class safely, though.